test/unit/migrations/settings/0002-migrate-settings-test.js

```diff
@@ -0,0 +1,230 @@
+                title: "Link Title",
+              },
+            ],
+          },
+        ],
+      })
+    );
+
+    assert.deepEqual(result, expectedResult);
+  });
+
+  test("migrate links with valid referrerpolicy", async function (assert) {
+    const settings = new Map(
+      Object.entries({
+        link_sections: "Section Name, Section Title",
+        links: "Section Name, Link 1, #, blank, Link Title, no-referrer",
+      })
+    );
+
+    const result = migrate(settings);
+
+    const expectedResult = new Map(
+      Object.entries({
+        link_sections: "Section Name, Section Title",
+        links: "Section Name, Link 1, #, blank, Link Title, no-referrer",
+        sections: [
+          {
+            name: "Section Name",
+            title: "Section Title",
+            links: [
+              {
+                name: "Link 1",
+                url: "#",
+                target: "_blank",
+                title: "Link Title",
+                referrer_policy: "no-referrer",
+              },
+            ],
+          },
+        ],
+      })
+    );
+
+    assert.deepEqual(result, expectedResult);
+  });
+
+  test("migrate links with invalid referrerpolicy", async function (assert) {
+    const settings = new Map(
+      Object.entries({
+        link_sections: "Section Name, Section Title",
+        links:
+          "Section Name, Link 1, #, blank, Link Title, invalid_referrerpolicy",
+      })
+    );
+
+    const result = migrate(settings);
+
+    const expectedResult = new Map(
+      Object.entries({
+        link_sections: "Section Name, Section Title",
+        links:
+          "Section Name, Link 1, #, blank, Link Title, invalid_referrerpolicy",
+        sections: [
+          {
+            name: "Section Name",
+            title: "Section Title",
+            links: [
+              {
+                name: "Link 1",
+                url: "#",
+                target: "_blank",
+                title: "Link Title",
+              },
+            ],
+          },
+        ],
+      })
+    );
+
+    assert.deepEqual(result, expectedResult);
+  });
+
+  test("migrate", function (assert) {
+    const settings = new Map(
+      Object.entries({
+        link_sections: "Design, Get inspired!|Code, Learn new things!",
+        links:
+          "Design, Design process, #, blank, Learn the basics|Design, Blog design, #, self, What makes for a great blog?|Design, Photoshop tutorials, #, invalid_target, Photoshop for beginners|Design, Design trends, #, blank, Stay on top of the current trends!|Code, Wordpress, #, blank, Wordpress code examples|Code, Tools, #, blank, Tools that will make your life easier!|Code, Tutorials, #, blank, Just starting out? We'll guide you through the basics",
+      })
+    );
+
+    const result = migrate(settings);
+
+    const expectedResult = new Map(
+      Object.entries({
+        link_sections: "Design, Get inspired!|Code, Learn new things!",
+        links:
```

Comment:
I'm not deleting the settings so that we have a way to recover should this massive migration goes wrong.